studio/src/windowManagement/contentWindows/ContentWindow.js

```diff
@@ -198,14 +198,19 @@ export class ContentWindow {
 		if (this.#preferencesButton) {
 			throw new Error("A preferences button has already been added.");
 		}
-		const button = new Button({
+
+		/**
```

Comment:
I just realised this type annotation is redundant because it gets inferred from the constructor's parameters.

Reply:
Good catch!